### PR TITLE
Update Feature Contribution Calculation Samples

### LINKED ISSUE #3233: Clean up FCC Documentation
The current documentation for the Feature Contribution Calculator is not showing up on the docs site, and needs minor changes.

Related to #1209

### PR DESCRIPTION
**Pull request by rogancarr**
This PR cleans up the samples for FCC and creates a new one specifically for calibrated learners.

Fixes #3233

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                .Append(mlContext.Transforms.NormalizeMeanVariance("Features"));
+
+            // Fit the pipeline.
+            var transformer = transformPipeline.Fit(data);
+
+            // Transform the data.
+            var transformedData = transformer.Transform(data);
+
+            // Define a linear trainer.
+            var linearTrainer = mlContext.Regression.Trainers.Ols();
```
Comment (zeahmed) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs]: >mlContext.Regression.Trainers.Ols() [](start = 32, length = 35)

Can we not combined it with the pipeline above? Is there any specific reason for doing so?

Comment (rogancarr) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs]: I'd like to compute the feature contributions on the transformed data, so I separate this out into two steps.

---
In reply to: [273270793](https://github.com/dotnet/machinelearning/pull/3241#discussion_r273270793) [](ancestors = 273270793)

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext(seed: 1);
+
+            // Create a small dataset.
+            var samples = Data.GenerateData();
+
+            // Convert training data to IDataView.
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+
+            // Create a pipeline to concatenate the features into a feature vector and normalize it.
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs]: This doesn't look a part of FCC. May we start with raw data?

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            // Now we train the model and score it on the transformed data.
+            var linearModel = linearTrainer.Fit(transformedData);
+            // Print the model parameters.
+            Console.WriteLine($"Linear Model Parameters");
+            Console.WriteLine($"Bias: {linearModel.Model.Bias} Feature1: {linearModel.Model.Weights[0]} Feature2: {linearModel.Model.Weights[1]}");
+
+            // Define a feature contribution calculator for all the features, and don't normalize the contributions.
+            // These are "trivial estimators" and they don't need to fit to the data, so we can feed a subset.
+            var simpleScoredDataset = linearModel.Transform(mlContext.Data.TakeRows(transformedData, 1));
+            var linearFeatureContributionCalculator = mlContext.Transforms.CalculateFeatureContribution(linearModel, normalize: false).Fit(simpleScoredDataset);
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs]: Is it possible to do `Fit(null)`? Having `simpleScoredDataset` is a bit confusing.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+            //  Bias: -0.007505796 Feature1: 1.536963 Feature2: 3.031206
+            //  Label: 1.55184 Prediction: 1.389091
+            //  Feature1: -0.5053467 Feature2: 0.7169741
+            //  Feature Contributions: -0.7766994 2.173296
+        }
+
+        private class Data
+        {
+            public float Label { get; set; }
+            public float Feature1 { get; set; }
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs]: You have spaces between public fields in the other sample below.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContributionCalibrated.cs [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+                        Feature2 = (float)(rng.Next(10) * (rng.NextDouble() - 0.5)),
+                    };
+
+                    // Create a Boolean label with noise.
+                    var value = bias + weight1 * data.Feature1 + weight2 * data.Feature2 + rng.NextDouble() - 0.5;
+                    data.Label = Sigmoid(value) > 0.5;
+                    yield return data;
+                }
+            }
+            private static double Sigmoid(double x)
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContributionCalibrated.cs]: One line function needs to use `=> 1.0 / (1.0 + Exp(-x))`.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs (hunk 3 above)
Comment (rogancarr) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs]: It is confusing, but we cannot use `null`. We need to pass in something with a schema.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs (hunk 2 above)
Comment (rogancarr) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs]: I wanted to show that FCC works on featurized data, not on the original columns, so I wanted to make it explicit.

**Pull request merged**